roles/nfsserver/meta/main.yml

```diff
@@ -14,4 +14,4 @@ galaxy_info:
       versions:
         - bionic
 dependencies:
-  - role: "common"
+  - role: deploy_user
```

Comment:
Can we use the new pattern where we put the dependencies in the playbook roles stanza, rather than the meta/main.yml? 

This will help us move towards having less redundancy in our runs and having both runs and tests run a lot faster.

Reply:
it seems to me like if you changed the playbook to have:
```yaml
  roles:
    - role: roles/deploy_user
    - role: roles/nfsserver
```
then that task would not fail, because once the nfsserver role ran, the deploy_user role would already have run

Reply:
Yes, if we remove stuff from `meta/main.yml` we need to add it in the playbook.

Reply:
The idea is to include the dependencies under `roles:` in the playbook instead of under `dependencies` in `meta/main.yml`. We'd need them in both the operational playbook and the testing playbook for each project.

Reply:
If the playbooks have `- deploy_user` listed in the `roles:` section, this task would succeed.